test/decoder/decoder_native_test.exs

```diff
@@ -10,8 +10,8 @@ defmodule Decoder.NativeTest do
     assert {:ok, file} = File.read(in_path)
     assert {:ok, decoder_ref} = Dec.create()
     assert <<frame::bytes-size(7469), _::binary>> = file
-    assert {:ok, _frames} = Dec.decode(frame, decoder_ref)
-    assert {:ok, [frame]} = Dec.flush(decoder_ref)
+    assert {:ok, _pts_list, _frames} = Dec.decode(frame, decoder_ref)
```

Comment:
It would be best to verify timestamps as well

Reply:
Yes, of course, I've purposely marked this PR as draft because I'm also going to add timestamps handling in the encoder and few tests as well.